spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jdbc/JdbcTransactionManagerAutoConfiguration.java

```diff
@@ -47,17 +46,17 @@
 @ConditionalOnClass({ JdbcTemplate.class, TransactionManager.class })
 @AutoConfigureOrder(Ordered.LOWEST_PRECEDENCE)
 @EnableConfigurationProperties(DataSourceProperties.class)
-public class DataSourceTransactionManagerAutoConfiguration {
+public class JdbcTransactionManagerAutoConfiguration {
```

Comment:
That's a breaking change as the name of the class is public API.

Reply:
I did consider changing only the configuration to return a `JdbcTransactionManager` instead of a `DataSourceTransactionManager`. I can still do that and revert this rename and only keep the implementation change?

Reply:
Reverted the rename and only kept the config change.

Reply:
Thanks but I explicitly asked you to not act on it 

>  I've added a note for myself in the review, no need to act on it now.

I might have to ask you to undo that based on further analysis.

Reply:
No problem. I assumed you meant the change from `DataSourceTransactionManager` to `JdbcTransactionManager` and not the change in the name of the configuration. If I need to change it again or add a migration path of sorts please let me know.